Kepler-RoboVetter-DR25.cpp

```diff
@@ -396,7 +396,8 @@ int main (int argc, char* argv[])
 
       // If it is the first iteration, save the major and minor flags.
       if(m==0)
-        data[n].disp_string = disp_pcfp + " " + to_string(data[n].not_transit_like) + " " + to_string(data[n].stellar_eclipse) + " " + to_string(data[n].centroid_offset) + " " + to_string(data[n].ephemeris_match) + " " + data[n].comments;
+        // data[n].disp_string = disp_pcfp + " " + to_string(data[n].not_transit_like) + " " + to_string(data[n].stellar_eclipse) + " " + to_string(data[n].centroid_offset) + " " + to_string(data[n].ephemeris_match) + " " + data[n].comments;
+        outfile << disp_pcfp << " " << data[n].not_transit_like << " " << data[n].stellar_eclipse << " " << data[n].centroid_offset << " " << data[n].ephemeris_match << " " << data[n].comments << "\n";
```

Comment:
Why did you add a new line ('\n')?
 I think it doesn't exist in the original to_string.